Gruntfile.js

```diff
@@ -38,12 +37,10 @@ module.exports = function( grunt ) {
 		'vendor/*/*/*.yml',
 		'vendor/*/*/.*.yml',
 		'vendor/*/*/tests',
-		'vendor/ampproject/common/phpstan.neon.dist',
-		'vendor/ampproject/optimizer/bin',
-		'vendor/ampproject/optimizer/conceptual-diagram.svg',
-		'vendor/ampproject/optimizer/phpstan.neon.dist',
+		'vendor/ampproject/amp-toolbox/bin',
```

Comment:
the `.phpcs.xml.dist` file n `ampproject/amp-toolbox` can be excluded.

```suggestion
		'vendor/ampproject/amp-toolbox/bin',
		'vendor/ampproject/amp-toolbox/.phpcs.xml.dist',
```